dpnp/tests/test_fft.py

```diff
@@ -926,6 +927,16 @@ def test_rfft_1D_on_2D_array_out(self, dtype, n, axis, norm, order):
         expected = numpy.fft.rfft(a_np, n=n, axis=axis, norm=norm)
         assert_dtype_allclose(result, expected, check_only_type_kind=True)
 
+    @pytest.mark.skipif(not has_support_aspect16(), reason="no fp16 support")
+    def test_float16(self):
+        a = numpy.ones(10, dtype=numpy.float16)
+        ia = dpnp.array(a)
+
+        result = numpy.fft.rfft(a)
+        expected = dpnp.fft.rfft(ia)
+        # check_only_type_kind=True since Intel NumPy returns complex128
+        assert_dtype_allclose(expected, result, check_only_type_kind=True)
```

Comment:
```suggestion
        expected = numpy.fft.rfft(a)
        result = dpnp.fft.rfft(ia)
        # check_only_type_kind=True since Intel NumPy returns complex128
        assert_dtype_allclose(result, expected, check_only_type_kind=True)
```